package.json

```diff
@@ -15,6 +15,7 @@
   },
   "devDependencies": {
     "eslint": "8.22.0",
-    "eslint-config-next": "12.2.5"
+    "eslint-config-next": "12.2.5",
+    "sass": "^1.54.4"
```

Comment:
to se dela automaticky kdyz das `npm install <some_package>` ja jsem dal `npm install sass` a to samo updatne `package.json`.

POZOR! az si to natahnes k sobe tak musi dat command `npm install` aby se ti stahnul ten package to slozky `node_modules`

Reply:
Tohle slouží k tomu, abychom mohli využívat sccs?